css-animationworklet/Overview.bs

```diff
@@ -182,18 +182,44 @@ partial namespace CSS {
 };
 </xmp>
 
+
+
+Animator {#animator-desc}
+========
+
+An <dfn>Animator</dfn> represents an animation instance that is running on animation thread. Animators
+are identified by a unique name and determine how the animation progresses its keyframe effects
+given current input time. <a>Animator</a> instances live in {{AnimationWorkletGlobalScope}} and
+each one is associated with a {{WorkletAnimation}} instance. An animator can only be instantiated
```

Comment:
(Just for my own clarification, I suppose a `WorkletAnimation` is the main thread instance?)